src/core/Akka.TestKit/TestKitBase_AwaitConditions.cs

```diff
@@ -307,15 +218,8 @@ protected static bool InternalAwaitCondition(Func<bool> conditionIsFulfilled, Ti
             return InternalAwaitCondition(conditionIsFulfilled, max, interval, fail, null);
         }
         
-        /// <summary>
-        /// Async version of <see cref="InternalAwaitCondition(System.Func{bool},System.TimeSpan,System.Nullable{System.TimeSpan},System.Action{string,object[]})"/>
-        /// </summary>
-        /// <param name="conditionIsFulfilled"></param>
-        /// <param name="max"></param>
-        /// <param name="interval"></param>
-        /// <param name="fail"></param>
-        /// <returns></returns>
-        protected static Task<bool> InternalAwaitConditionAsync(Func<bool> conditionIsFulfilled, TimeSpan max, TimeSpan? interval, Action<string, object[]> fail)
+        /// <inheritdoc cref="InternalAwaitCondition(Func{bool}, TimeSpan, TimeSpan?, Action{string, object[]})"/>
+        protected static ValueTask<bool> InternalAwaitConditionAsync(Func<bool> conditionIsFulfilled, TimeSpan max, TimeSpan? interval, Action<string, object[]> fail)
```

Comment:
Although internal, this method is protected and not private, we can't change the return value.
Add CancellationToken support.